igvc_description/launch/spawn_jaymii.launch

```diff
@@ -1,17 +1,34 @@
+<?xml version="1.0"?>
+
 <launch>
 
-    <!-- Vehicle pose -->
-    <arg name="x" default="0"/>
-    <arg name="y" default="0"/>
-    <arg name="z" default="0"/>
-    <arg name="roll" default="0.0"/>
-    <arg name="pitch" default="0.0"/>
-    <arg name="yaw" default="0"/>
-    <!-- Spawn the vehicle. -->
-    <node name="spawn_platform" pkg="gazebo_ros" type="spawn_model"
-          args="-file $(find igvc_description)/urdf/jessi.urdf
-                -gazebo_namespace /gazebo
-                -urdf -model Jessi
-                -x $(arg x) -y $(arg y) -z $(arg z)
-                -R $(arg roll) -P $(arg pitch) -Y $(arg yaw)"/>
+  <!-- Vehicle pose -->
+  <arg name="x" default="0.0"/>
+  <arg name="y" default="0.0"/>
+  <arg name="z" default="0.5"/>
+  <arg name="roll" default="0.0"/>
+  <arg name="pitch" default="0.0"/>
+  <arg name="yaw" default="0"/>
+  <!-- accelerate lidar with GPU -->
+  <arg name="gpu" default="false"/>
+  <!-- Spawn the vehicle. -->
+
+  <param name="robot_description" command="$(find xacro)/xacro $(find igvc_description)/urdf/jessii.urdf.xacro gpu:=$(arg gpu)" />
+
+
+  <!-- uncomment below line to change to using the old robot -->
+  <!--<param name="robot_description" command="cat $(find igvc_description)/urdf/jessi.urdf" />-->
```

Comment:
Change this from uncomment to an env / arg?

Reply:
I'm going to leave it. I don't think anyone is going to really use the old way the robot would launch.